scripts/docker-run.sh

```diff
@@ -8,7 +8,7 @@ fi
 
 DOCKER_HOST=${DOCKER_HOST:-localhost}
 CONTAINER_NAME=${CONTAINER_NAME:-timescaledb}
-DATA_DIR=${DATA_DIR:-$PWD/data}
+DATA_DIR=${DATA_DIR-$PWD/data}
```

Comment:
It needs to be - because DATA_DIR can be blank. None of the other variables can be.